### PR TITLE
opencv: update to 3.4.2.

### PR DESCRIPTION
**Pull request by jnbr**

### CODE REVIEW

File: srcpkgs/opencv/template [hunk 1]
```diff
@@ -14,7 +14,7 @@ homepage="http://opencv.org"
 license="BSD-3-Clause"
```
Comment (maxice8) [inline srcpkgs/opencv/template]: license= above homepage=

homepage= is now on https

Comment (jnbr) [reply srcpkgs/opencv/template]: changed it. Added ck skip to not waste another 5 hours of travis-ci.

Comment (maxice8) [reply srcpkgs/opencv/template]: forgot to push ?

Comment (jnbr) [reply srcpkgs/opencv/template]: forgot to add

**Pull request merged**